nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -60,11 +64,13 @@ def triage_patient(self):
         status = self.triage_task.execute()
         if status == Task.ERROR:
             raise NotImplementedError()
-
-        if status == TriageTask.TIMEOUT:
+        elif status == UserInputs.WD_TIMEOUT:
```

Comment:
Is there any way to standardize the outputs so that the mission planner does not depend on the constants defined in the ros bridge? Can some standard status output be given from the triage_task based on the user input

Reply:
I could duplicate the constants inside the task with interface communication task so the user inputs are evaluated and then a standardized output is returned.